curp/src/client.rs

```diff
@@ -331,7 +335,7 @@ where
                 Err(e) => {
                     warn!("wait synced rpc error: {e}");
                     // it's quite likely that the leader has crashed, then we should wait for some time and fetch the leader again
-                    tokio::time::sleep(retry_timeout).await;
+                    tokio::time::sleep(retry_timeout.current()).await;
```

Comment:
Do we need to advance backoff next retry here?

Reply:
Sorry I have merged current and next_retry so that it automatically increment every time.